### ISSUE TITLE
Subject case rule is ignored

### ISSUE THREAD

**Issue by grrinchas** [Gherciu/commitlint-jira#48]
Hello, 

I have found that overriding `subject-case with [2, "always", "sentence-case"]` no errors are reported.

For example, 
```
git commit -m "EX-12: subject case" -- should throw error
git commit -m "EX-12: Subject case" -- should pass
```
Maybe we can add new rule for this? or fix ti?

Thanks.
